layers/state_tracker/cmd_buffer_state.cpp

```diff
@@ -1810,21 +1811,21 @@ void CommandBuffer::GetCurrentPipelineAndDesriptorSets(VkPipelineBindPoint pipel
     *rtn_sets = &(last_bound.per_set);
 }
 
-void CommandBuffer::BeginLabel(const char *label_name) {
+void CommandBuffer::BeginDebugUtilsLabel(const char *label_name) {
     ++label_stack_depth_;
-    label_commands_.emplace_back(LabelCommand{true, label_name});
+    debug_label_regions_.emplace_back(DebugUtilsLabel{true, label_name});
 }
 
-void CommandBuffer::EndLabel() {
+void CommandBuffer::EndDebugUtilsLabel() {
     --label_stack_depth_;
-    label_commands_.emplace_back(LabelCommand{false, std::string()});
+    debug_label_regions_.emplace_back(DebugUtilsLabel{false, std::string()});
 }
 
-void CommandBuffer::ReplayLabelCommands(const vvl::span<const LabelCommand> &label_commands,
-                                        std::vector<std::string> &label_stack) {
-    for (const LabelCommand &command : label_commands) {
-        if (command.begin) {
-            label_stack.emplace_back(command.label_name.empty() ? "(empty label)" : command.label_name);
+void CommandBuffer::ReplayDebugLabelRegions(const vvl::span<const DebugUtilsLabel> &label_regions,
+                                            std::vector<std::string> &label_stack) {
+    for (const DebugUtilsLabel &label_region : label_regions) {
+        if (label_region.begin) {
+            label_stack.emplace_back(label_region.label_name.empty() ? "(empty label)" : label_region.label_name);
```

Comment:
I could just now rename `DebugUtilsLabel::label_name` to `DebugUtilsLabel::name`, no need to reming `label_`
```suggestion
            label_stack.emplace_back(label_region.name.empty() ? "(empty label)" : label_region.name);
```